src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/index.js

```diff
@@ -212,16 +212,28 @@ function AnalyzeAreasContainer(props) {
       newSelectedOption === WDPA_OECM_FEATURE_LAYER;
 
     const getLayersToToggle = () => {
-      const formerSelectedSlug = selectedOption.slug;
+      const formerSelectedSlug = selectedOption?.slug;
       const newLayerCategory =
         newSelectedOption === HALF_EARTH_FUTURE_TILE_LAYER
           ? LAYERS_CATEGORIES.PROTECTION
           : undefined;
 
-      let layersToToggle = [
-        { layerId: formerSelectedSlug },
-        { layerId: newSelectedOption, category: newLayerCategory },
-      ];
+      let layersToToggle = [];
+
+      if (newSelectedOption !== 'clear' && formerSelectedSlug !== undefined) {
+        layersToToggle.push({ layerId: formerSelectedSlug });
+        layersToToggle.push({
+          layerId: newSelectedOption,
+          category: newLayerCategory,
+        });
+      } else {
```

Comment:
The clear and the selection is not working correctly here in relation to the layers. Two examples:
- Go to the analyze areas
- Click on the clear button. The radio is deselected ✔️ 
- Click on the Places for half earth future. The future layer is not selected ✖️ 
- Click on the Administrative Boundaries. Now the future layer is selected ✖️ 

----

- Go to the analyze areas
- Click on the Places for half earth future. The future layer is selected ✔️ 
- Click on Protected areas. The protected layers are selected ✔️ 
- Click on the clear button. The places for half earth future layer is still selected ✖️